rewrite-core/src/main/java/org/openrewrite/GitRemote.java

```diff
@@ -279,7 +279,7 @@ private String repositoryPath(RemoteServerMatch match, URI normalizedUri) {
                     .replaceFirst("^/", "");
         }
 
-        private static final Pattern PORT_PATTERN = Pattern.compile(":\\d+");
+        private static final Pattern PORT_PATTERN = Pattern.compile(":\\d+/");
```

Comment:
what if the org is a number 🤔

Reply:
Well at least github supports number-only orgs. (https://github.com/1234)